python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -7,34 +7,45 @@
 {% if traffic_policies is arista.avd.defined %}
 !
 traffic-policies
-{# Global options #}
-{%     if traffic_policies.options.counter_per_interface is arista.avd.defined(true) %}
-   counter interface per-interface ingress
-{%     endif %}
 {# FIELD SET #}
+{#     -- L4 PORT MANAGEMENT -- #}
+{%     if traffic_policies.field_sets.ports is arista.avd.defined %}
+{%         for field_set_port in traffic_policies.field_sets.ports %}
```

Comment:
We can use natural_sort here.

s1-leaf1(config)#sh run | sec traffic
traffic-policies
   field-set l4-port ASERVICE-DEMO
      10, 20
   !
   field-set l4-port BSERVICE-DEMO
   !
   field-set l4-port SERVICE-DEMO
      10, 20, 80, 440-450

Reply:
done